google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITQueryWatchTest.java

```diff
@@ -638,4 +650,11 @@ private static void debugMessage(StringBuilder builder, Map<String, Object> data
       }
     }
   }
+
+  private DocumentReference setDocument(String documentId, Map<String, Object> fields)
```

Comment:
If you change the fields parameter map value to `?` it removes the need to have the explicit `Object` casting above.
```suggestion
  private DocumentReference setDocument(String documentId, Map<String, ?> fields)
```

Reply:
... but then I can't show off my casting skills! Fixed.

Reply:
Haha